common/authorization/default_authorizer_test.go

```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 	"go.temporal.io/server/common/config"
+	healthpb "google.golang.org/grpc/health/grpc_health_v1"
```

Comment:
```suggestion
	healthpb "google.golang.org/grpc/health/grpc_health_v1"

	"go.temporal.io/server/common/config"
```

server imports in a separate section